This fixes multiple instances of:

	_Warning: Package '<data_pkg_name>' is absent from the packages configuration.
--- a/litepcie-2024.04/setup.py	2024-04-16 22:43:46.834348505 +0200
+++ b/litepcie-2024.04/setup.py	2024-04-16 22:44:09.857501663 +0200
@@ -1,7 +1,7 @@
 #!/usr/bin/env python3
 
 from setuptools import setup
-from setuptools import find_packages
+from setuptools import find_namespace_packages
 
 
 with open("README.md", "r", encoding="utf-8") as fp:
@@ -30,7 +30,7 @@ setup(
           "requests"
         ]
     },
-    packages                      = find_packages(exclude=("test*", "sim*", "doc*", "examples*")),
+    packages                      = find_namespace_packages(exclude=("test*", "sim*", "doc*", "examples*")),
     include_package_data          = True,
     keywords                      = "HDL ASIC FPGA hardware design",
     classifiers                   = [
